ui/src/app/views/workspace-edit.tsx

```diff
@@ -722,8 +773,27 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
             </div>
           </div>
         </WorkspaceEditSection>
-        <WorkspaceEditSection header='Request a review of your research purpose'
+        <WorkspaceEditSection header='Request a review of your research purpose for potential
+                                      stigmatization of Research Participants'
```

Comment:
text: I'm confused by capitalization of "Research Participants". It's not used anywhere else on the page and I don't think this is anything like a proper phrase or legal term we've defined elsewhere.

Reply:
karthik approved, changing to lowercase